client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -310,9 +323,39 @@ class CancelPurchaseForm extends React.Component {
 		const reasons = {};
 		const { translate } = this.props;
 		const { questionOneOrder, questionOneRadio, questionOneText } = this.state;
+		const { productSlug: productBeingRemoved } = this.props.purchase;
+
+		// get all downgradable plans and products for downgrade question dropdown
+		const downgradableJetpackPlansAndProducts = [
```

Comment:
Maybe you know this already: the `planHasSuperiorFeature` can determine if a plan is a "sub-plan" of another one as well. But future plans may not that much overlap, so this feels brittle. The map seems like a better solution indeed.

Reply:
Oh actually, no I didn't know about `planHasSuperiorFeature`, thanks! TIL! 👍  🙂